packages/react/src/constants/SharedPropTypes.js

```diff
@@ -37,7 +37,7 @@ export const HotspotContentPropTypes = {
       thresholds: PropTypes.arrayOf(
         PropTypes.shape({
           comparison: PropTypes.oneOf(['<', '>', '=', '<=', '>=']),
-          value: PropTypes.any,
```

Comment:
There could be an argument for keeping any here and adding an exception, but I added all the various types used currently.